File: backend/src/main/java/com/claims/manage/domain/InsurancePolicies.java
package com.claims.manage.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Data
@Document(collection = "insurance_policies")
public class InsurancePolicies {
    @Id
    private String id;
    private String type;
    private int amount;
    private int maxClaim;
    private List<String> description;
}

File: frontend/src/app/model/insurance-policies.ts
export class InsurancePolicies {
    id:string
    type:string
    amount:number
    maxClaim:number
    policyTerm:number
    description:Array<string>
    constructor( id:string,
        type:string,
        amount:number,
        maxClaim:number,
        policyTerm:number,
        description:Array<string>){
            this.id=id;
            this.type=type;
            this.amount=amount;
            this.maxClaim=maxClaim;
            this.policyTerm=policyTerm;
            this.description=description;
    }

}
